Cargo.toml

```diff
@@ -25,12 +25,11 @@ wasmer-clif-backend = { path = "lib/clif-backend" }
 wasmer-runtime = { path = "lib/runtime" }
 wasmer-runtime-core = { path = "lib/runtime-core" }
 wasmer-emscripten = { path = "lib/emscripten" }
-
-[target.'cfg(not(windows))'.dependencies]
```

Comment:
@xmclark 
Does this `not(windows)` basically mean that llvm can't be enabled via a feature here?

With llvm being a workspace member, are you able to run `cargo build --all` successfully on Windows?

Reply:
Why would we want to run `cargo build --all`? It doesn't make sense to build non-binary crates individually.

Reply:
It’s useful in development to ensure changes in upstream upstream projects compile against all the other projects. For example, a change in runtime-core should be check against all other projects. It is also useful to run other commands that affect all projects in a workspace like `cargo clippy —all` or `cargo fmt —all`

Reply:
@bjfish correct. It would not work at all on windows if the attribute was still there.